ccd-definition/AuthorisationCaseField/CareSupervision/barrister.json

```diff
@@ -506,13 +506,167 @@
     "UserRole": "[BARRISTER]",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "orderToSendOptionCount",
```

Comment:
I noticed a similar field that is just named "optionCount", could we possibly reuse that field rather than creating a new one?

Reply:
I've crated orderToSendOptionCount so that can be placed on the eventData rather than be mapped on the root CaseData. It's an extra field, but it worked out better for the business logic. If I used the common, I'd have had to pass the whole casedata in a lot of methods